index.bs

```diff
@@ -391,6 +391,10 @@ Note: The group is <a href="https://github.com/webmachinelearning/webnn/issues/8
 
 Implementers of this API are expected to be familiar with the <a href="https://gpuweb.github.io/gpuweb/#security-privacy">WebGPU Privacy Considerations</a>.
 
+Ethical Considerations {#ethics}
+===================================
+
+The Working Group has started documenting ethical issues associated with using Machine Learning on the Web, to help identify what mitigations its normative specifications should take into account. This work currently happens in a dedicated <a href="https://github.com/webmachinelearning/ethical-webmachinelearning">GitHub repository</a>.
```

Comment:
At some point, this ought to become something like

```suggestion
The Working Group has started documenting ethical issues associated with using Machine Learning on the Web, to help identify what mitigations its normative specifications should take into account. See [[ethical-webmachinelearning]].
```

But that'll have to wait for your draft to get into https://github.com/w3c/browser-specs or for you to add a [custom biblio entry](https://tabatkins.github.io/bikeshed/#biblio-custom) for it, and it'd be reasonable to wait for the WG to publish it as a [Draft Note](https://www.w3.org/2021/Process-20211102/#draft-note) before doing that. What you have seems fine for now. Thanks!

Reply:
The repo or the doc name might still change, so I intentionally did not want to codify the names yet. We’ll convert this into a biblio reference when we’re more solid.